src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: right triangle, area: " + getArea() + " sq"
```

Comment:
Use only eng in messages/code. Replace 'sq' with 'sq.units' to keep the code universal and consistent.